packages/normalizr/src/index.js

```diff
@@ -155,41 +155,44 @@ const getUnvisit = entities => {
   const cache = {};
   const getEntity = getEntities(entities);
 
-  return function unvisit(input, schema) {
```

Comment:
this whole block until denormalizeAndTracked  is actually just returning `[previousReturn, cache]`

Formatting changed each line due to the tabbing over.